docs/src/pages/docs/comparison.md

```diff
@@ -8,44 +8,53 @@ title: Comparison
+|                                              | React Query                            | SWR [_(Website)_][swr]     | Apollo Client [_(Website)_][apollo]   | Relay [_(Website)_][apollo]           |
+| -------------------------------------------- | -------------------------------------- | -------------------------- | ------------------------------------- | ------------------------------------- |
+| Supported Query Syntax                       | Promise, REST, GraphQL                 | Promise, REST, GraphQL     | GraphQL                               | GraphQL                               |
+| Supported Query Keys                         | JSON                                   | JSON                       | GraphQL Query                         | GraphQL Query                         |
+| Query Key Change Detection                   | Deep Compare (Stable Serialization)    | Referential Equality (===) | Deep Compare (Unstable Serialization) | Deep Compare (Unstable Serialization) |
+| Query Data Memoization Level                 | Query + Structural Sharing             | Query                      | Query + Entity + Structural Sharing   | Query + Entity + Structural Sharing   |
+| Bundle Size                                  | [![][bp-react-query]][bpl-react-query] | [![][bp-swr]][bpl-swr]     | [![][bp-apollo]][bpl-apollo]          | [![][bp-react-relay]][bpl-react-relay]|
+| Queries                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Caching                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Devtools                                     | ✅                                     | 🟡                         | ✅                                    | ✅                                    |
+| Polling/Intervals                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Parallel Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Dependent Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Paginated Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Infinite Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
```

Comment:
Another instance where, in the new beta comparison, we also have "Bi-directional Infinite Queries", which from what I understand is usually *possible* in most libraries, but not officially supported and thus, one I'd like to verify on Relay's end.

Reply:
I'm not sure what Bi-directional Infinite Queries mean

Reply:
Its a term for describing an infinite query that you can cursor/window in both directions. Imagine you start with a window that shows the data for page 6. From there you could scroll-up and load in more results from page 5,4,3,... or scroll down and load in more results from 7,8,9... and so forth. This is bidirectional infinite querying, where you can technically go in both directions from a single root cursor/index of data view.

Reply:
Another good example would be something like a log window. New logs are coming in above you, but only get loaded as you scroll up and old logs are available below you if you scroll down. This would presumably use cursors to "paginate" the data in chunks in either direction you go.

Reply:
Relay can do this with the paginationContainer or in experimental with the usePaginationFragment but only forward and backward not batches between it as far as I know.

Reply:
https://relay.dev/docs/en/pagination-container#createpaginationcontainer